code/modules/reagents/reagent_dispenser.dm

```diff
@@ -13,9 +13,7 @@
 	var/lastrigger = "" // The last person to rig this fuel tank - Stored with the object. Only the last person matter for investigation
 
 /obj/structure/reagent_dispensers/attackby(obj/item/I, mob/user, params)
-	. = ..()
-	if(I.is_refillable())
-		return FALSE //so we can refill them via their afterattack.
+	return FALSE

```

Comment:
This now prevents all attacks.
You should have kept the original code and just moved `. = ..()` underneath of the `return FALSE` directive.